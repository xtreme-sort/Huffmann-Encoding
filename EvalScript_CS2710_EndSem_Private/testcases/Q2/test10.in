DecodeMessage
((((((A,F),Y),(N,y)),(a,3)),(I,(((E,M),H),Q))),(w,i));
011000011010110000000000001001110110010000010001000000101111000011000000000001011001100110100110011010011001001101100100011001100011100110010010011111000000011011000110110010110101000100110100000111000100111000000100111000001000000000000011000001100000001101100110100001000001101000001011001000001011001000110110010010011001000001000000000010011011010000100000001011011101100101100101100100000100010000010111000000011110000000011010011000001001101101000001011101001101011000011110100010001001110110001100011011000011111011010000010000100000000011000000011010001000001000100110010111011010010000110000001100000000011011001010000110110010000000000010110010000101001100000010000000000001011001001000100001001100000100000000110000000000001001100100010000010000010000000100110000100111011001011010001101110000000000100100010011100000001110001101001110111011000011010000011100011000001000100011000001001000100000100110110010100110000100000010000000001000010011000011010001001100000100000010000100010000001000111001100000111100110000010000100000100011001111000000001101100001100100000000010000100110100011010000010000100011011001001100010011101100001000100100110010110010000101110001101110001010000110001101110010001001110001000010100111000100001010000001000001100000000000000111000000000001000000000110000001110010000100111000010100110101010010000100000010110010111001101100100001000000000110110100000000000001100101000011001100000000011011001001100100000000110011000010000001001001100010001100010010000000011100100110100010001001001110011111000100000000100111000001110011000101000000000100111011000001000001110110000110100000000000101101011001000001001000000101100011011000000110001100000101100111001001010000110110000001100011011000000110110101100101100001101000110000000111011000001110010001100100110000011000001011100001000000110011001001110110010111011101100100100110000110000110000110000000000111011100100000100100001101110000101101000010110010111001100100001111110110011110011101000000011011001000100001001111111000010110010000000111010000010000010110101100001111001101101000011000000011010001100011000101011011001011100011010001110011110010010000001011001010000100011100001001110001000001001001101100000100011100111000111011000010000000001000110011001001100000110010001001100101001100100011100111011110000100001000000000001011010111001001100100000001000000100000000010001100010110110110011010111011000100111011001000100110100100001101110110100001000010001001100100000000011011100000101110110100110001000011011000000101100101001110111000011100010000110110101101011100000001100000011001100011000101101000100000000000000111110000011100001011000000010110101100100010000001000010001001111001100000001000101100010100000110011000010011001011000011001001000001000100000000001001110110000000001001001100100001000000000000011001001101000011000000100111000100011000010001000100010011000000000000001100110100011010001000000100010000000000000000110011100100110010111001000001011100011000000000001111110000100110110000110010100111001100001011000100110110001100100000100100001100000100011110011011000000001100000101100111001100100111000100000000000100011011010111011110100001000100011010110001100000001101000000001001110011010011001011110001000010110000001111110000000000000000011000000000100001000100111110000010000101110100111001001100110110110000111000111010100000100000000000001101100001001100101110000010001101100010011001000011001001100101101001101100000001000010001000001011001000001011010110100111101010001001100001100000001011001100000100110011011100110000011001101011101100100011000010101010010000100000100000111011001011010111001011011001000000000000011010110101110010011001011100110110010110001101100001101000100000100000100011000000011000000000011000000000000101000000000110000000001101101001000000101000001011010110100000011011010000100100110101000000101100000010011010110101100100010110110011111011101000011000001110000010011001000000000100110000100000000000010000100011000001001000000011011101100001100000100001110000010011100000100010001101101010001100010000110000101100001000000100011011000000001011001001001100011011000000011000011011001011101110001100101101101001000000001000001011010001100000101110000011010001101101001000000100011011010001001101011000100001101100101101100110011001110111011100011010001101101000001011100010011100011001100000000110000000000001001001111000100000010110001001100001100101110110100011001001100000000100010001000100000000000010000100011100110101100100100001101100101100100011100110000000100010011101101100000010110000110010100000100000010000100110010000000001101001100001100101000000101110010010000000000010011011010001100010001000011111101100100010000010011000000011001011000000000010101010100000010011000110110100100110010110111110001100000101100100000100000100000101110101100100000010110000000111000001000001010011110011100010001101100001110001101100110101000000101100111010011000100011000000000101000000001100100000100001001101001101011001010010010110000000100100100000000100110010010011000011001110010001001000010000111001100111001011011010010110010000100001100100000010110100011001100010011010011001100100111011010110010010011011000000000000000010001110011000000001100000011000100000101100001101001000000000100000100001011010000000111001011000010010000100000000011011011101100011000000011011100100110100101001100000000001100101110000000010010011100011001000111001000011001101110100010010011000001100001110001000110000000001001100100000001100000010011000000010010011001001000110010011001000100000010010010001111011100000001101000100110000110000110010110101000110001100000111010000100001100011011001000010110010001100001011010001100100111011001000110001100110000010100000010000101001100000010000000000110110100001000010110100000001100001100000100000100010011100011011010001011011010001001001100100011001000100000000010011111000001011010001100000000000001110110100110000101100000011100011011100000000101100000101000000100011011010000000100010000100011011010000000011011010000110011001011100111001000001100010010000001011001000010010011000100100000010110010010100000010110111000000011001011001011101001100000000101100001110100000101001101010011001100110010110010000101000000010000001000111101110010000000000101000011000000000100010011010001000001000000000000010000110110010101001100101101000010110011100011001010001100000000000000010000110010011001000100011011000000001000110000010000101110010000000111001100000100000010111011000011100100110100110000100011000000001100010011100100000010000011100011001000000100000101110110000001001000011011001011000000000011000011001100111000100100000000001101100110001011011010111000100001100110100001101001100000100000010110100000100001001011110111100000100011000100001000011011111011010001100110111000000011001011000011001001101000001000010110101101001000000001110000101110100110010110100100001000001000000010011001010001100110110101110110000011000100100000000010010001000001000010000000000000111000011100000000011100110000110000111000110100011011101100110000000000000001000001001000001001101001110110011001100100100100110000000010100110101001100100001000010111011100110000100010001000110001101001100001101000001011000000111100010011011100100100000000111000001111101001110000000110000000010110101011011011001001000100110000110000000100000000111001001101110000000101101100000000000011100000011101100101101011001100001000000000110011011010000111011000011010010000111100010001110001100001000000100000000000101100100011000010000010010001000011011101100001100001101000100110010110010001101101000011000001001100001011010110010011011001000000001100001000001000001011001000110110000100110100100110100000000001000100000100000011100111000111101100101110110000110000100001011000000000100001001100001101011010000101101000110001101110010000001001001110110001000010000000010011001011010100011011000100110101110001100000000000100000100011001101000110110101100101100101101000101001100001100111000000000110000010001101101000000100001000111000011001000000100000100110110010111011000000100110111110111001100011011000110110000110010000110000010001000000100000000000000000001100001100001000000101101011000100010011001110110101100101101000110000100000101100100110011000100011000000000001010010001000000100001000100110010000000010000100110010001100000100100010011000100110100011001100000000000000100010000100001100011100101001101011101001110100000101101000000100111001000010100001100000000011000001001100100011000110000101100000000101001100101100000011000010000010110000110100000000000011001101100100001000010100011000100100000010110000110100000001100100010000000110110000000100101101000001100100110000001100110001000000101010001000001000000010000000000010000101001100000011011010011000100000010111010100000010011000001100001100110100111000010001111010011000001000011010100010000000011100000100001011000000010000100001011010010000010010010001101101011001000010000000110101100001101000000000110010011000000000011101101000001100111001100000100000001100000010001101100001111101000100001100001110000100001000001000100001000011001101000000001001101011001011101110001010000110011000011000000100000001000110000010010000000001100010110001110000001000110000001000001000100010011011000001000001000000000000000001001101101011010000000100001000000000010001100001011000000001011110000000110110101110110010010011010110000010000001011001001001001100001110100100110100110001111001000111100011011010000010110010000000110100011011111000100010000110100110000001001111100010000000000100110011001100000100010000101100100110010001110111100110110100100000011001100101100100010001100000001100100110110100010000110110010010011001011000011101101000100000010100010001000111101110000000000010110010000000110010001001101000001010000100110000001000000000100000010011000111000011000100111110001000000100110001101101011000000000000100011110011010111000010001100000000000100000000011001100000100011000100110100010100110010110010110010010000110100001001011011001001101101011111000010000000000011110011000011001011001000110100110010000000000010011000000001011011011100011100110010110010000110001001100100110000011100110111011100011000000100001101110010001010010000000001101100100011001001101000010000010110010001111011010001101001100100011011100101000010011000011001000101000001000100100000101001100000001000100100011000010110000001101101100011000010000000001100010011000011001011010000010110000001001100100001011010111000100000001100011011000000001000001011010010011001011000010000100101100010011001000010001111000010001000001011010111100100010011100000010000010001001100101010011010000100000100001011101101000100000010010110110000110100011001000100011000000011101001100100000001100000000000011100001001110001001110010011000000000010100100001001110000100001000001000110110000110101001111000010011000010001101100001101000001011001000110110010000011100100001100001001101101000011111011010011011100010000010110100001011010110100110110101000010011001011101100001001110110000001000110001000001011001000110000000000111100110101100000010110000010000100011001100000011000110000010101001000011011001100000100000001101011000000110110000011000000000110110101110110011000111000001000000000010001111000001011101110011010011000000010111000100001100010000010110101101110101001100001100000000100110110010010000001000110100110000001101110000101110100010001110000010000000101101100100000110011010000111000010000010000100011010011000011010001000001000010001110000010100111000110011001001110111110001000010000001011000110000000110010111000110110000110000001101101000001011100110000011110011101000001001001100000010000001011001000010001001000111101001101110110001100000100111001100110011000011101101001110011010011011010001101100100110110100000101100110011010000100000001100100100111001000010000001001000110000000011011010110101100000110001100011001111001100100010001100100000010100000000011000100000000010000001000110110100011001101100001110110000000010000000001001011110110010110000111001001100110100000101000011011000010011001000011010011001000001011000011110001000000100000000010000110111000000000001110001001100110011000100000001101001111001100100010001001101010001001100111011010110000110000000000001101111100000100011000110000010110010110101101010000101100010011000011001010010001010011101100011011000011000001100000101100010011000001001000000100010011001000001000000011001011010000000110100000000000100111001100101100001111000000000000100111100000001110110001100100001010110111000000110001000011010000110000100010000010110100010100000100010001001000000001100100000001101000100001001100100110000001000000100000000010011010000000001000000100000101110000100000100001000110100011000001011101110110010000000011011100010011010110100011000010110010111100110010001001000000100011011100010110001101100101100000011010000000100100011000100011010000000011100100000010100001100110110101101000110010001101101001001100111001011000000000100011000000010011101001101011100000010011101100100000000001110110100110111000001011001011100100001000100000011100000001100111000110110010010011001011010011000110000011101100000010001010011010110000111010011010000100001110101100110011011100001011000000100110000000000000100000001000000100110110110001011000000011100001011010110010001010000110000101001100101100100011011010000000110110011001011100100001101110001001110111000010100000000000010110110000110111000000000001011010011000101100011011100000000000001000000100100011010011001000000011010111000001010011010000010000000000011101100100110110000100111000001000100001110011001000010110100000000100110110000000000001011111000000001000100111001111100010000010001001101000000000000110000000010110010110001001000101101110110010100001000000101110000010001111000000000000000110001000100110000010011010001100000101101001101100000000100101110000110000000011110001001100000011000100110001001100100001011001000010000101110011000111011100110111011001000001000000010001000001011010110101100001100101101001000010001000010001111001000001000100111000111110011101101000001000010011011010001000010000100000100100100001101000110110000011100001010011001001000100111000010000000000010001001101000010110000000000000001110001000111000000001100101100001101110011110000100110000100101100110000000011110000000000101100000000100001000110000100010000000011001011001110000100011000010110010110110001100011001000010011010110100110000010000010110101100001000101110000001011000000110001101101000110111011010001100000110000111100000011010011000000100110101110000010000100011011001110111010000001011001000100000100000001100101101011001110110010110000001100011011000100000000000001100001000110110010110010110010111110011110111100011011000001001110011000100001100110000010010011000011000001010000010011110111000101100110000111011100001000100111001100011010000001001110011001000010111000000100000101100101110001100011000110010011000000100100001101100000010000001000000011001000101101100110001000011001001000000000110010000001001101100100110001101100011011000000001100110000110010110110100110100000000010000110000101100000001001111011101000001011001000101111100000100110110000010000001011000110000100110011100110100000000000001100000000000011000000010000010110000110000110011000011011001011010010011000000110000000000010110101101000100111000000000001011000000001011100010011001001101101001101000000101100010010001100000000101000101000011110011000111000100001101101010000010111000100001000000101100111110000010010010100000101100000111111000100100000011101110110000011100110010110000001100110110010110101101100001100001011101100100000101100000000000010001100110000001101100100000101110010010000110010000010110101101000011000100001000010000110000000100100110101100010000000001000110011011010000101101010110111011100011010011001000110001101100100100011000001011001000010111011000110111000010110000001100000010000011000100000100110010100110100001001010000110110000110100011001100001000010010001100100100001000011010011010010000100011011000110001000110110010010010010000000011010110011011011010111001100000100100111001100011011001110011011111001000000000001011001011001110110010000100011110000000000101000101000011000001100001000011000110001000100011011100110110010100011001100000101110010011010100110100110111001000011000001001010001000000000000101100100001000000011001111000110111011010000010110000110100010010100011000100000100000101000011000000010000000000000010100110010001001100100011110110000001001101011001011010011000110001100011000010001100011011001000111110000000011001110100111001000011000000011010000010100011011010100110010000100001000010110000110000110001001100000000111001100000000110010000000000001000001000001011011000000100100100000100011100001000011000000000110001101101000100011000110000001111000010100110101010110110000010110100000111000001000011000001011000110110010001001101011011000010011010110100110000010001000011011100010001000000101100001000000010001100000100000001000000100010001101110000010001101011000001001100001001100100111000001000100000100100110000000111001100001000000000110100001001100010000100100001001001100000110011000010000100000001100000101000000101100000101101100100011011010001111000010010000011000110110010000100100001100010000011001100000100110010000101100100001110011001100011000001011010001000010010100000000010111100110100001000000101000000100001100110100001000000110110000110000111011001010110001101001100000001000100000011001000000101100000010000100000000001100010000010011011000000001000100110100001001110011000101000011000000010000100110101001111010000010010000100110100001100100001001100100000110001100011000000000110111000010100010100010000000011000100110010100010001101001100000000000110000000011011010001100000100111101100101101011000000100011100001101001100001100101001100000100101000100001000001011000010010000000111110000010000010110010110101111000010000110110011001101000100110010000000011000010110010000100100111001010100100010000110001000000001110110010000111011010110010000010110010001100100111011101000011000100110010110000011011001000011100011000110111000010001100010100001010010011100110111000000100001100100000000110101100011000001110100110000001000110100110101100001100100000000000101000001000101101110110010000000001000111101100011010001001100000011101100000100001011000001100100000100100111011001100110100011110000000100000000110100001100001100100001100000100001000010110010110101110000010010000100000000010011001000001001001100000010011001001001000001011001000100001100000000001010100011000000011010001100010010000110110100010111011000000000001011000010000000000110110000010000001011001011000000000000110000000000000001111010000000100001101100100001001101101011001000000011100000001110110100100110100100000100000000000000000000000000000101110000000000000000100000000110110000000000110101101001000100101110000000011000011010000101100000010010011010111000110111011000001001110100100001100000001001110011000100110000000010000100000110000100110010001100010001100011000010110010110011011000111000101011000010000000000010110000000000001000100000010110100000110001100011011001000000011010110010000010111000100110000100111000111100100000101100110001000000100001011000100110100001010011001001101110010000000100000000100111000001000001110001100110100100110010110100100110000000000000010101000100001100011001000011001000110000010000000011011100100011001000110000101101011001000010000010000101110110110010000100010000111000100001001111101001101011000011100011000010110000000011100001110000100000000001110110010001001110100001001001100101110110010001001100000011000111001100101000100110000010000000000100111011100010011001100000000110000111000001011100001000110001000010000010010011010001101100001000011001100010100001100001010101101100001100000001000010000010001011001111000001000100001100100110010110010001101100001100011011010010011000111001100101000100100001000001000001100000000111000000011011001010011010001000000001100000001011001000110000010001000000000001011100100111011010011000010001101011011000011100010100110010100000000111011101101011101100001100000000000000100110110000010011001000100001001000011011000011000011000000010001110000000110110101110000010011010011001010011010100001110100111001000000100100110010001101100000100111111010011000000010110010010011010000100000000000100011011000100001001101011001011100000001110111000110000010111000000000000100110000000000110101100011001000010000000011110001100000001001101000110110010010001101100100100010111101110011010011000011011001100000100111111000000000011011101100101101011010001101001110110010010000000001000110001110011010000000011000001011000000110000100100001100011100110110110100010101101100001100101110000010000000000111000100100000010110000000000010110011100011000010010000000010011000011010011000110001101000011011011111010000100010011000100111001100010001001111111011101100011000000001100111000001011100110001001000000000001000010000010110010110111000100100000101101011010110001100001000010010000110000010111000110000010111111100100000101000000100110001001101000000011010001000010000110000100010011000001000000100100110000010110010110101010011010000101100000000000000001100000110000011001000110000111000000011000001111000110000000001001001100100000100010000010110100010011000011001010010000010100000011000000000011110010000110010011101100000010011010000010011110000101110000010001000000110011001000100011010001110000110000000010011101001101000000011100110001101001101100000110110000010001000000001001001101011001001001101001001100111000110001001110000101110111110010000000011001011000000011101110111011010000000110000110100110110101110111110000010011000000011010000100100110000110010010011111010011010100110101100101000011010011101100100100101001101011010100110100100110011100000101100001100100001101000110001100110111011000000001011001000110001001110111000000001000010000110011011000011000001100000101100000000101100110010001000001001101100010011010110100010011100001001001101001111011000011001000000001100000100110010001101100100110111011000011001011010100110000110101110000100010010011010011100110100100111001100110110000111011001110010011000000100001000011010000001000001001000100001100000101110111010001000010000010111011000000110010011001001000010100010100111011010000001001100100011001000001011011000010011001000100010010000100101100001000011101100001100000001000101101101011100001000100000000000011101100101101011010110100000000100100001100100001100100000010010000010011011001000101001100100001000110100110010110010010000010100111000110111100000000001100000000100110100011011000011101100100000100010011010100000010110101101001001100111011010110010011011001
Quit
