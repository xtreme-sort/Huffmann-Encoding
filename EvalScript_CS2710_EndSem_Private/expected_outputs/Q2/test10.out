EHEANQMFNFQwyAFMwH33IM3My3ywMaQiAHw3MHIaHFiNQAwQFAAE3AHwHYFwwYMFMyMaMFAY3HYAIiQMMMFNYQAQwAH3F3HFQIHEQwwaaQEiyEQiHFYAyAHNYNMQHayAiAyMIyMAFMYIENAFMaaNEaAEANMNYFAIEIQMHyQAYaaQAQyIQQEHFiyFN3FaaY3MIEIFANNEHNEaFYNFywE3i3FYFy3iA3EMANNHyIYYyM3NQEIaIMMYQyQNwyyQaaQNNwQNNwFFwAAQAFAyAiwINQYIHIwINFMQ3MYAyHAAMIy3AyM3aAMwNFaEw3NIAQaHNaIQ3iwaAIQFi3NwAaQEaYiEHAFHMFaFEiEyyFMiaIwyEyyEyHMEHyAQE3wI3aE3FQYAi3aQMQQMaEEEEAQQaYayQYHYMQ3ayiiMiwQIAiMNNQiiYMAQIYFHEQwHwwyAHyyNwiMQyI3wQwIIFMIN3wNQNYaHwYywQywiYAN33aE3aaMIMywQQwNNAYHQaMAIFAN3NiHwHQEwQMNHayQHYYNMAyQFQH3NyENiawQQYiNyHHQAEy3yNiIaAAQiFiYEYHMNFYNQwEYNiNwYwEIMEMaYNANQEAwIMYAAM3IyAwQN3YNaaEAFwHyIaFNAAy3wIMQaYQyAFiiwNHwyawQ3YEwHw3aYayFyi3EFwYMi3aQNAYyHQQwwNa3IiyAHAaQ3IMQwaNiAiiiAAFwAaNaQiFYQIQaMwiEQywwwYAAHwNMQFyEwMYwIMH3EYYNYMFHH3iIwaEEYMwY33Q3FwHQMyYIwwINYFiMHQaiMAAHHQaMQ3MEiEHNYFyAEAEANwA3AyHaFIYHHAiHYaHIFENHHMNiMiiQIyFiF3aAaEIAFYyFaAiQEEaywY3wYN3HI3NyYEIFyEFMaEiEYwyMQQyaiHaAIYHyFQFww3HaFyHNHEwyMHwMwQQQyI3HFQNQy3A3AAwIQwaFEwEMQHyaEFNaaAFYywHMayMMywEYNQHwFEMIYAwNMAyIEMIFQaIAY3HyNayiiMNY3AMEAIwwwwF3yHaMHiiyFMFFFQIiaFEYiFFIQwQN3EQyMwwwFMiIEw3ANwAMFY3IHMIIIiAIIIAIMaEMiaaINywMiaiHaiaNyaFHy3NH33aQHMaHwAAN3wEFwFwaYEHaAaYYHAQaiYaNA3HiEiAHiaHawEAMQAaIQya3wIy3QIaIE3YiN3ANMAENEYaMa3aMNFaI3iQAHNEEMHI3yFiINyyMYMyYHyaQMyy3FIFYIENAyHYYHAEEaYNQyHNiHNIMyaaAaQiFHyAAQH3YEyw3QAaiFIFyHAIaN3HA3HYwMQ3wIYwaIFMYaEwIFMawFHiAMMQIEFEQIYIHIMwMMYIAwFyiQaANwyANaHNYAAIyMIwMHYMiyaw3AANyaMN3EFyFYQaAiwEaFQEQaH3YyA3NQaFFiyaFFQENIyMEAEMwQNIAyMwaiHQNy3IyIEaFHFYaiiQwYyNNyQiHy3QAMEM3IYYHHaAQYQIMHaNYAIMI33HQE3NIAaIaYYAAQYiAywEEQyI3QMwAAaYaY3IQMwMaIEFIHIMYYQQ3YNa3yIEHFEyiNHiaIAQFiiIQAEFHIiHwIIaEEYA3wIHiAIiEAywFiMHMwNA33HYiEHayiN3w3YAwAYMyYFaayQEEHNMMyHYwY3YHM3MA3YFFMyEIHaHAYNYAiwQyiMQEEINiANNEHHYHyyQaFaQEwNAIMHI3EwHQyAFFy3I3HMMHNwEMiAyFyHAwN3wyaFF3MQENHiiQ3yEiEMYwYNFAAAEEIFHEwaMiHMHyYFM33N3AFIIaFYNMAaNMyFaaEwHy3AAaaNyywIwHQIQIYHAwQaNwyAyF3a3yYEFIMEyYFEHAAi3MYYI3NIFEHAMNAiEYaiIYwIEy3NFIwaYAIAYYIEyH3NFQIwF3Fwy3IQYyiIEayIwaAQFYEYYYHaYaI3HMYAHEHAyaEAQHFwQ3FAEN3EQiIayYiYYFNNy3IAIHMQQNwy3YwFAI3FaA3NiywFyAwYNaHwYFAAF3HHAINAN3YEFQwAiHQMaHEaFMaIEQIIH3yia3iyHFMAHyQiNayIENQiNANMwEaaNiaMywii3HaFwMMN3AM3HNyMaMEQHNFIaa3iQAFMAMNHFINENAaF3ywyNQiNF3yHEAN3i3IiwN3AFAy3FyNHNwMMMayINIiM3HQiYAFiwEMMyIMAF3AaiHiywMMYwaM3Fi3QQyAwyQaawIA3MyaHYFMyiHyIMyQawNEMNwYNIYIEYNI3YEyHw3YAyNEMHFENMYHQNAiyEFFHaMEINIiNMYyiYNYHQwIaQAwYNMIwHYFYQHNFaiEHyaa3AQIMAEAywNQNQaEAIwINQYYFyEHIQwNEI3EHFMyMFiayY3HYiiH3QNYHYHH3HINMQEIQEN3NYMyAYiwHENiFYy3AiyFIwIyMwYAHEyE3AyHQMw3wYAYyiFQQ3IEYQNyNYHHiIwEEF3MaFyIEyQYQIa3wYAIiMFwHYiYFYyIEHNYYywYIQy3aQQiNNFEiAMQyEEyHFQ3FiwQIYaENFMYNI3iIHiEiF3wMwEQH3wH3HyM3HFMwHYAMaQaNFa3A3HHE3yy3i3aa3aFIA3NAaFyHy3EQEFANIiiMEQaMwwYIyEIMYwwMFEQwaFANyQAFiNMwEwAiIQwMNaHIaMiHEEAyQiFyyFMHHINiNEMIIawQEiEE3FEwEaIFNMFAMHAHAF3wMEQwAF3iAQEiaNwiQAiNyIyYNYHNwYNaIAMAHNNM3AwFANHANFFQYFYyI3FQQMA3QNHHyYMQwMNIFyQNiyMEyIAwI3N3IAQaFIy3HHya3HaMiaiAN3AIQIHQAwQMAYiH3QFMQaNaFiAMiyMaMH3yFiENawHEQIHYYiIi33QYENEAYAIF3HwaiAQYHMNwyYIMMyHAHwMQayQNQQYIAFHwyQAFH3NiyQAAIFa3IMAHQFIHFAFiM3EIQFNywMYHAaHwAFQiAaaQ3iwaYNHAAiAaiaiNIaiQMINFQFyiAAyNaEaHyFH3EFaiwyA3iNEyNEwMYMYYQ3ywiwHiMFAIaYHHEMHaNaN3iaYNQyiwQHFY3HNNNYaIyI3E3wNwMaaQYAFNHYEAAiwa3wAMEHi3iY3Yai3A3iAYEFYyYNAMMiYyYMHw3yaNHH3FFHEIaiwFEyyHyQHyFwyiAiIENHQFYyMiQIFMNYAMHMiMEyyEwAAiYyMMMQi3iQw3EaQ3Ny3FaEEawY3iQNi3YiQYNQ3yIF3wMYQAwYMQyyyaENIyENFAMNiMwayaIA3aF3M3yEiEFwEMHwwHANyYEY3iQIYMNiiwY3EaFEiY33wHAAEAyAIYEEMwyMHaEyAFHHNQAFEFQNM3H3IFEwI3Aawawyi3ywayHIYQNNFMiiFaIwYE3iiNIFiQE3wMEy3MHHwyYQMFEAN33AiMFQaIyaYHHYwaNNyAIIHEwAa33HYHIiQQyIMyyMa3FMYQEiQYEyAwYwaY3awHYawyEHy3YYa3aINyIHaN3EiN3MaIIAHMwiHQ3FaQ3yMi3QiaAYMMiMYyiAYIawyFwNyyNa3Q3MI33FQaHIH3QayFawaAFMYAMiw3QHFEHNIw3NYFIyAIAAIwMNMyiENHMH3yyyYyyMyiwAMiIQayAHFI3HIMYYYEEEwEFi3A3aAFFFHwFaIYywNyAyyHN3yAiiYIHIwiEaiIYiFYwYEiMNHHwNHH3FNyQNaFEIAw3FAIFN3QFyIiF3Y3aQFNYaEYi3YAyINEwNINIE33YYAEawFEaiMyHyiYaYw3MYayNYwEaMYMYi33yFHNNIwAaii3INFIFYwHYAiEEQMIiyIEYNFwIFENNAyNY3EFNHY3wEwwyAINHIQwwYaNHYwINMFw3yAyQYIawaAEwMIa3IEA3A3HyF3iMHEN3wyIEMIEaIwaNYEIIAiiwYFMHQwNyMwHNMA3YMYaQawwIayNAQMYiHMFMyaQQIyNME3MYiyyQYyNwNwIQ3QAwyaAHEiFiIEN3IHEMAFIYNiQMAN3iEiIaEywiFYE3aYaQMwHyiAIAHYwyayFYYMHQFaNAaMFaENMaIYMNyAYIw3AHyNIyHNiwiAFEIAyEaFMEAyAAyiIAwyMY3HMAQAQHaHaYAAAAFQAAYA3EAHHaaIiwAEHYENIHQyQEaQIIyAIQ3NEFYFwNMyN3yYMMwiywawiYAFEANaFHFw3yMAHMFQNEIQyiaYMwaFYEwHYIM3QaAwAIQFFiy3IIMHaEAFIwayyaya3FA3Qa3a3YHMYFYQHwINaywaNQiIHEQyYEFiYiYAYiMNQINIMQMNEyywMIaEaANQQNMwAEQFQYyNNYaHyEIy3NwyYIwiEEYYFNi3iFNyaMMyEEiHaEiwMIaINYFwAQAHwIwHNAEYMyFNAYQaQH3YyIiEQNwMIAQQHQEEAF3EaMNNIyEEEYywAiHQF3IMIHIywwQaFaMyEaQiwwEYMaHYAFyEwNHMQAQQyFQAAwEAHEiaNAQw3AIHyMa3MaaiiQ3IEHwEaQiwAyQMHHyIQMaAa3ywHA3FEyYayywHwiIawiEMQFAYiNIFEAai3w3YaAIEH3yyIyHiiINaEwQ3NaQiiQEiA33wYQ3NIAYYFMHiNIYHHEiYYayFQyFQiiaYIF3NHAHNNyYNEaFaEaiaiIwwHYEAAE3FwI3YiAE3iyANIMFNYHNEMIIYIFwAyiayaQENHF3iYQFNFwMN3I3wyAaQIHAQ3yIHwYwiFNAIIHMaHaMiyNQYQQiaAMEYiQQHAEH3HQQiF3AHYaEMaQiIHIHMIyIQMaIwHHIHaMiFEMYww3y3QEFMyNQQAaNy3EE3FEFMwIaY3EwHHNQYaH3iEMA3F3a3M3QEMHIEHQYNIH3wHaQ33EQMiaENNyIFFaayFQQIaNYQEyaMaNwawQHAwMyaYHwNMNaINIiYYiEEYNiHQYNAFiMHHHAaIyayaFaY3MNwMYyIMMaYIQyQwAyAaHyEQMFNHIFHHaMiHM3M
